default/scripting/techs/construction/FORCE_ENERGY_STRC.focs.py

```diff
@@ -16,68 +16,68 @@
             accountinglabel="CON_TECH_ACCOUNTING_LABEL",
             priority=AFTER_ALL_TARGET_MAX_METERS_PRIORITY,
             effects=[
-                If(
+                EffectIf(
```

Comment:
this seems awkward

Reply:
`ConditionIf` ?

Reply:
internally it's called `Conditional` ... Can't think of anything better than that.

Reply:
Changed to it.